terratest/test/kgb_basic_app_test.go

```diff
@@ -18,8 +18,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Basic ohmyglb deployment test that is verifying that associated ingress is getting created
-func TestOhmyglbBasicAppExample(t *testing.T) {
+// Basic kgb deployment test that is verifying that associated ingress is getting created
+func TestkgbBasicAppExample(t *testing.T) {
```

Comment:
I guess it should be `TestKgbBasicAppExample` in order to comply with naming conventions.